Windows/Gopher/Gopher/Gopher.cpp

```diff
@@ -82,6 +86,9 @@ void Gopher::loop() {
 		return;
 	}
 
+	if (CONFIG_DISABLE_VIBRATION)
+		_vibrationDisabled = !_vibrationDisabled;
```

Comment:
Add brackets around the if statement body. 
Also, I think you need to add a sleep command of some sorts here. I haven't tested it, but If you hold down the *vibration disable* button, it will toggle vibration on and off at the same frequency as the mouse movement (144 times each second). 

Please correct me if I'm wrong